src/VirtoCommerce.ExperienceApiModule.Core/Infrastructure/SchemaFactory.cs

```diff
@@ -22,8 +22,9 @@ public SchemaFactory(IEnumerable<ISchemaBuilder> schemaBuilders, IServiceProvide
         }
 
         public bool Initialized => _schema.Value?.Initialized == true;
+        public string Description { get; set; }
 
-        public IFieldNameConverter FieldNameConverter { get => _schema.Value?.FieldNameConverter; set => _schema.Value.FieldNameConverter = value; }
+        public INameConverter NameConverter { get => _schema.Value?.NameConverter; set => _schema.Value.NameConverter = value; }
```

Comment:
Can we got `NullReferenceException` in setter if `_schema.Value` is `null`?

Reply:
I don't think so. When Value will be called the first time the instance of Schema will be created.